resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,97 @@
+
+{% endtab %}
+
+{% tabs %} { % tab title="iOS" % }
+
+  * **Verify that Server cerificate is configured in Campaign instance.**
+  
+      Ensure that server certificate (.pem file) is successfully set up in Campaign instance. This can be verified in Campaign instance under following section Campaign -> Administrator -> Channels -> Mobile App (AEP SDK). Launch mobile app and check push channel settings, it should say Android key set up success. It should look like:
+  ![alt text](https://github.com/shivam-tomar-sde/aep-sdks-documentation/blob/push-troubleshooting-document/.gitbook/assets/campaign_ios_cert_configured.png "Configured push certificate in Campaign.")
+
+{% endtab %}
+
+{% endtabs %}
+
+{% tabs %} { % tab title="Android" % }   
+  * **Set up Android app for receiving push notifications: Android app should contain following for receiving push notifications:**
+
+    * Create Firebase Messing Service and add it to Android Manifest file. [Refer](https://firebase.google.com/docs/cloud-messaging/android/client).
+  
+    * Verify that user is opted in for receiving push notification. This can be verified in Griffon. [Refer](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#ensure-user-opt-in-for-push-in-adobe-analytics).
+  
+    * Generate push token for app using Fire Base Instance Id class. Set push token to AEP SDK by calling Set [Push Identifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference#set-up-push-messaging). Verify that Push token is successfully synced with Identity service in Griffon. [Refer](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#verify-push-token-sync-with-the-experience-cloud-identity-service).
+
+    * For app to receive push notification it’s push token should be present in Campaign instance mapped with correct Experience cloud id(mid). This can be verified by following 2 steps:
+                 
+    * Verification in Griffon: Verify that the user id is correctly set. [Refer](https://aep-sdks.gitbook.io/docs/resources/troubleshooting-guides/troubleshooting-push#confirm-that-the-user-id-is-correctly-set).
+
+    * Verification in Campaign Instance: Verify that app’s push token is present in Campaign instance mapped with correct Experience cloud id(mid). This can be verified by logging into Campaign instance and then navigating to following section: Campaign -> Administrator -> Channels -> Mobile App (AEP SDK). Launch your app and check under mobile application subscribers. It should contain list of all Subscribers of app. Verify mid and push token for your app are present there. This can be verified on the page shown in below screenshot. **Experience Cloud ID** and **Registration Token** for the user should be present here.
```

Comment:
Make this steps 8-10:
"8. Verify that app’s push token is in the Campaign instance that was mapped with the correct Experience cloud id(mid) by clicking Campaign -> Administrator -> Channels -> Mobile App (AEP SDK). 
9. Launch your app and, under the mobile application subscribers, review the list of all Subscribers of app. 
10. Verify the mid and push token and ensure that your app is displayed and that **Experience Cloud ID** and **Registration Token** for the user is displayed.